packages/jellyfish-transaction/src/script/defi/dftx_token.ts

```diff
@@ -25,3 +26,39 @@ export class CTokenMint extends ComposableBuffer<TokenMint> {
     ]
   }
 }
+
+/**
+ * TokenCreate DeFi Transaction
+ */
+export interface TokenCreate {
+  symbol: string // ---------------------| VarUInt{1-9 bytes}, + n bytes
+  name: string // -----------------------| VarUInt{1-9 bytes}, + n bytes
+  decimal: number // --------------------| 1 byte
+  limit: BigNumber // -------------------| 64 bytes
```

Comment:
```suggestion
  limit: BigNumber // -------------------| 8 bytes
```

Big number is not 64 bytes